mlog/default.go

```diff
@@ -6,6 +6,7 @@ package mlog
 import (
 	"encoding/json"
 	"fmt"
+	"os"
 )
 
 // defaultLog manually encodes the log to STDOUT, providing a basic, default logging implementation
```

Comment:
The comment is not longer consistent with the code

Reply:
Good catch :+1: